<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldEditorWindow</class>
 <widget class="QMainWindow" name="WorldEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="world_editor.qrc">
    <normaloff>:/icons/ic_nel_world_editor.png</normaloff>:/icons/ic_nel_world_editor.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="LandscapeEditor::LandscapeView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="worldEditToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newPrimitiveAction"/>
   <addaction name="loadPrimitiveAction"/>
  </widget>
  <widget class="QDockWidget" name="treePrimitivesDockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>3</number>
     </property>
     <property name="spacing">
      <number>3</number>
     </property>
     <item row="0" column="0">
      <widget class="WorldEditor::PrimitivesView" name="treePrimitivesView">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>250</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="loadPrimitiveAction">
   <property name="text">
    <string>loadPrimitive</string>
   </property>
  </action>
  <action name="newPrimitiveAction">
   <property name="text">
    <string>newPrimitive</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LandscapeEditor::LandscapeView</class>
   <extends>QGraphicsView</extends>
   <header>../landscape_editor/landscape_view.h</header>
  </customwidget>
  <customwidget>
   <class>WorldEditor::PrimitivesView</class>
   <extends>QTreeView</extends>
   <header>primitives_view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="world_editor.qrc"/>
 </resources>
 <connections/>
</ui>
